querier.go

```diff
@@ -54,6 +54,9 @@ type Series interface {
 
 	// Iterator returns a new iterator of the data of the series.
```

Comment:
Worth to mention that we iterate here over raw samples? `SeriesIterator` does not tell that.